<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportCADDialog</class>
 <widget class="QDialog" name="ImportCADDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>880</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>931</width>
     <height>541</height>
    </rect>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>5</y>
      <width>531</width>
      <height>55</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="PBtn_ImportCAD">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>5</y>
       <width>91</width>
       <height>47</height>
      </rect>
     </property>
     <property name="text">
      <string>导入到块</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PBtn_cancel">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>5</y>
       <width>151</width>
       <height>47</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PBtn_Turninto">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>5</y>
       <width>91</width>
       <height>47</height>
      </rect>
     </property>
     <property name="text">
      <string>转进</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PBtn_OpenNewfile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>151</width>
       <height>47</height>
      </rect>
     </property>
     <property name="text">
      <string>打开新文件</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>数据转换</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>坐标：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_coor1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_coor2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>-&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>旋转：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_rotate">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>90</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>180</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>270</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>911</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
